File: dart/1295.dart
class Solution {
    int findNumbers(List<int> nums) {
        int c = 0;
        for (int i in nums)
            if ((i > 9 && i < 100) || (i > 999 && i < 10000) || i == 100000)
                c++;
        return c;
    }
}

File: python3/1295.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        c = 0
        for i in nums:
            if (i > 9 and i < 100) or (i > 999 and i < 10000) or i == 100000:
                c += 1
        return c
